src/render/processor.rs

```diff
@@ -38,7 +38,8 @@ pub trait AudioProcessor: Send {
         inputs: &[AudioRenderQuantum],
         outputs: &mut [AudioRenderQuantum],
         params: AudioParamValues,
-        timestamp: f64,
+        current_frame: u64,
+        current_time: f64,
         sample_rate: SampleRate,
     ) -> bool;
```

Comment:
My personal limit for number of method arguments is five. We are at six now. I'm wondering if I should combine `current_frame`, `current_time` and `sample_rate` in a `GlobalScope` struct